src/Styleguide/Components/ArtworkGrid.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import { Responsive } from "../Utils/Responsive"
+import LegacyArtworkGrid from "Components/ArtworkGrid"
+
+export const ArtworkGrid = props => (
+  <Responsive>
+    {({ xs, sm }) => (
+      <LegacyArtworkGrid columnCount={xs ? 2 : sm ? 3 : 4} {...props} />
```

Comment:
While this nested ternary reads pretty well right here would you mind breaking this out a bit into a normal if / else conditional or switch? These are hard to unpack when returning to code after being away for a while.